src/bg/background.js

```diff
@@ -1,4 +1,9 @@
-chrome.browserAction.onClicked.addListener(function(tab) {
-    chrome.tabs.executeScript(null, {file: "src/inject/inject.js"});
-});
+"use strict";
 
+void (() => {
+    const api = typeof browser === "object" ? browser : chrome;
+
+    api.browserAction.onClicked.addListener(() => {
+        api.tabs.executeScript(null, { file: "/src/inject/inject.js" });
```

Comment:
This is the only change that is actually needed for WebExtensions (besides using `browser` namespace instead of `chrome`) -- the `file` relative path needs to start with a `/` to be calculated relative to the extension's root.